app/api/agent/route.ts

```
@@ -1,37 +1,31 @@
-  apiKey: process.env.OPENROUTER_API_KEY,
-});
-
 export async function POST(req: Request) {
-  const data = await req.json();
-  const image_url = data.image_url;
+  let data;
+  try {
+    data = await req.json();
+  } catch (error) {
+    return new Response(
+      JSON.stringify({ error: 'Invalid JSON in request body: ' + error }),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)